fennel/featuresets/featureset.py

```diff
@@ -74,13 +77,17 @@ def feature(
 ) -> T:  # type: ignore
     if len(args) > 1:
         raise TypeError(
-            f"Please reference to only one feature or one field at a time found : {args}"
+            f"Please refer to only one feature/field/expression at a time found : {args}"
         )
     if default is not None:
         if len(args) == 0:
             raise TypeError(
                 'Please specify a reference to a field of a dataset to use "default" param'
             )
+        if isinstance(args[0], Expr):
+            raise ValueError(
+                f"Default value cannot be set for an expression, use fillnull instead, found : {args[0]}"
+            )
```

Comment:
`error in extractor of {feature}; can not set default value for expressions, maybe use fillnull instead?`